src/domain/__tests__/config.test.js

```diff
@@ -0,0 +1,30 @@
+import Config from '../config';
+
+describe('Config', () => {
+  let env;
+
+  beforeEach(() => {
+    env = process.env;
+  });
+
+  afterEach(() => {
+    process.env = env;
+  });
+
+  describe('featureFlagShowCoronaInfo', () => {
+    it('should return true when REACT_APP_FEATURE_FLAG_SHOW_CORONAVIRUS_INFO is true', () => {
+      process.env.REACT_APP_FEATURE_FLAG_SHOW_CORONAVIRUS_INFO = 'true';
+      process.env.REACT_APP_FEATURE_FLAG_SHOW_CORONAVIRUS_INFO = true;
```

Comment:
The first one could be removed.  Or is this following the same logic as the test below and is just missing 
`expect(Config.featureFlagShowCoronaInfo).toEqual(true);` in between these two lines?

Reply:
I'm trying to check that both a boolean and a string value yield true, so yup, I'm missing one expect 👍